debian/patches/0069-add-tonemap-videotoolbox.patch

```diff
@@ -1807,6 +1850,7 @@ new file mode 100644
 +    { "tonemap_mode", "Tonemap mode selection", OFFSET(tonemap_mode), AV_OPT_TYPE_INT, { .i64 = TONEMAP_MODE_MAX }, TONEMAP_MODE_MAX, TONEMAP_MODE_COUNT - 1, FLAGS, "tonemap_mode" },
 +        { "max",      0, 0, AV_OPT_TYPE_CONST, { .i64 = TONEMAP_MODE_MAX },          0, 0, FLAGS, "tonemap_mode" },
 +        { "rgb",      0, 0, AV_OPT_TYPE_CONST, { .i64 = TONEMAP_MODE_RGB },          0, 0, FLAGS, "tonemap_mode" },
++        { "relative_luminance", 0, 0, AV_OPT_TYPE_CONST, { .i64 = TONEMAP_MODE_RELATIVE_LUMINANCE }, 0, 0, FLAGS, "tonemap_mode" },
```

Comment:
How about we abbreviate this new method to `LUM`? (IMHO the shorter the better in cli options - `TONEMAP_MODE_LUM` and `:tonemap_mode=lum`)

Reply:
My main concern is the ambiguity of the abbreviations. Unlike MAX and RGB which has a very clear and definitive meaning, LUM could be misinterpreted  as three terms are very similar: The luminance L measured in cd/m^2, the relative luminance normalized to the percentage of the reference white in linear space, and the luma Y' which is a  a weighted sum of nonlinear R'G'B' components. If we are going to make such abbreviation, we need a clear documentation.

Reply:
I've thought about this as well, especially the ambiguity between lumi and luma. Only clear documentation can solve this, otherwise we have to make the choice for the user by implementing the `auto` option and hide these specific methods behind.

If users use `ffmpeg -h filter=tonemap_*` to query, then things should be much simpler.
```c
    {     "max",       "Brightest channel based tonemap",  0, AV_OPT_TYPE_CONST, {.i64 = TONEMAP_MODE_MAX},          0, 0, FLAGS, "tonemap_mode" },
    {     "rgb",       "Per-channel based tonemap",        0, AV_OPT_TYPE_CONST, {.i64 = TONEMAP_MODE_RGB},          0, 0, FLAGS, "tonemap_mode" },
    {     "lum",       "Relative luminance based tonemap", 0, AV_OPT_TYPE_CONST, {.i64 = TONEMAP_MODE_LUM},          0, 0, FLAGS, "tonemap_mode" },
```

Reply:
I added help text to tonemap modes, which should be good enough to allow as to use `lum` as abbreviation